This file is part of MXE.
See index.html for further information.

Contains ad hoc patches for cross building.

From 359e279bcaa185972ff82fd4e7469bb71c3a6b93 Mon Sep 17 00:00:00 2001
From: MXE
Date: Sat, 30 Mar 2013 17:07:15 +1100
Subject: [PATCH] add extra cflags for static build in pkg-config file

This patch has been taken from:
http://liblqr.wikidot.com/forum/t-638530/static-cflags-in-pkg-config-file

diff --git a/configure.ac b/configure.ac
index eaaa3ae..9b761cc 100644
--- a/configure.ac
+++ b/configure.ac
@@ -141,7 +141,8 @@ AC_ARG_ENABLE([declspec],
 	  )],
      [declspec=true])
 AS_IF([test "x$declspec" = "xfalse"],
-      [AC_DEFINE([LQR_DISABLE_DECLSPEC],[],[Disable __declspec directives])],
+      [AC_DEFINE([LQR_DISABLE_DECLSPEC],[],[Disable __declspec directives])]
+      [AC_SUBST(EXTRA_CFLAGS,-DLQR_DISABLE_DECLSPEC)],
       [:]
       )
 
diff --git a/lqr-1.pc.in b/lqr-1.pc.in
index 5ced291..3babc99 100644
--- a/lqr-1.pc.in
+++ b/lqr-1.pc.in
@@ -8,4 +8,4 @@ Description: LiquidRescale seam-carving library
 Requires: glib-2.0
 Version: @PACKAGE_VERSION@
 Libs: -l@LIBRARY_SNAME@
-Cflags: -I${includedir}/@LIBRARY_SNAME@
+Cflags: -I${includedir}/@LIBRARY_SNAME@ @EXTRA_CFLAGS@
-- 
1.8.0.3

